test/utils.ts

```diff
@@ -121,7 +121,8 @@ test('getRepoPath', t => {
 
 	for (const [url, result] of pairs) {
 		location.href = url;
-		t.is(result, utils.getRepoPath(url));
+		console.log(result, utils.getRepoPath());
```

Comment:
Types detected that the url is not passed in. Internally, it reads from window.location.